Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -21,11 +21,13 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
         super.tearDown()
     }
 
-    func test_edit_when_there_are_helpers() throws {
+    func test_edit_when_there_are_helpers_are_setup_are_config() throws {
```

Comment:
nitpick: probably you meant something more like `test_edit_when_there_are_helpers_and_setup_and_config`?